tests/unit/test_slide.py

```diff
@@ -694,7 +701,14 @@ def it_knows_its_slides(self, tmpdir):
             slides = slideset.slides
 
         assert isinstance(err.value, FileNotFoundError)
-        assert str(err.value) == "[Errno 2] No such file or directory: 'fake/path'"
+
+        if os.environ.get("TRAVIS_OS_NAME", "linux") == "windows":
+            assert (
+                str(err.value)
+                == "[WinError 3] The system cannot find the path specified: 'fake/path'"
+            )
+        else:
+            assert str(err.value) == "[Errno 2] No such file or directory: 'fake/path'"
```

Comment:
I think that this statement should be replaced with something more general. For example the `errno` attribute on the error should be the same on all platforms. You will get WindowsError exceptions on Windows, but since this is a subclass of OSError the same "except OSError:" block will catch it. Windows does have its own error codes, and these are accessible as .winerror, but the .errno attribute should still be present, and usable in a cross-platform way.

Something like:
`OSError(errno.EEXIST, 'Error Message here')`

Reply:
Unfortunately, there is no `errno` attribute in `err`. Moreover, the error message is decided by the OS, it is not ours, so I cannot even try something like `"error message" in err.value`. Any suggestions?

Reply:
`errno` is in the std lib
so my huntch is 
```
import errno
assert str(err.value) == errno.EEXISTS
```

BTW here the reference https://python.readthedocs.io/en/latest/library/errno.html